package.json

```diff
@@ -63,7 +63,12 @@
     "@azure/msal-node": "^1.18.4",
     "axios": "^1.7.2",
     "tar": "6.1.9",
-    "glob-parent": "5.1.2"
+    "glob-parent": "5.1.2",
+    "lodash": "4.17.21",
```

Comment:
lodash doesnt seem to be necessary in the resolutions, its being fixed in another pr, merge main changes and validate